web-console/src/pages/streaming/inspection/[config]/[view].tsx

```diff
@@ -0,0 +1,165 @@
+// See the status of a input table or output view.
```

Comment:
this file was removed in latest main because of https://github.com/feldera/dbsp/issues/270#issuecomment-1672085294

should be deleted, also need to make sure rebase / merge was correct for the other files in pages too